src/main/java/dinkplugin/notifiers/KillCountNotifier.java

```diff
@@ -112,7 +136,7 @@ private void handleKill(BossNotificationData data) {
             .replacementBoundary("%")
             .replacement("%USERNAME%", Replacements.ofText(player))
             .replacement("%BOSS%", Replacements.ofWiki(data.getBoss()))
-            .replacement("%COUNT%", Replacements.ofText(String.valueOf(data.getCount())))
+            .replacement("%COUNT%", Replacements.ofText(data.getCount() + (ba ? " high gambles" : "")))
```

Comment:
Why does this even mention High Gambles? That's just a point reward you can buy.

Reply:
We don't know the penance queen kc & need to report *some* sort of count